taxi/admin.py

```diff
@@ -1,6 +1,10 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
-from .models import Driver, Car, Manufacturer
+from .models import (
+    Driver,
+    Car,
+    Manufacturer
+)
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)

Reply:
It's my mistake, of course if you import more than 2 -> create a new line)